sdk/core/core-client/src/interfaces.ts

```diff
@@ -502,28 +632,75 @@ export interface BaseMapper {
   constraints?: MapperConstraints;
 }
 
+/**
+ * Mappers are definitions of the Data models used in the library.
```

Comment:
NIT: consistent casing
```suggestion
 * Mappers are definitions of the data models used in the library.
```